wellcomeml/ml/bilstm.py

```diff
@@ -130,7 +131,11 @@ def residual_attention(x1):
         )(x)
         model = tf.keras.Model(inp, out)
 
-        optimizer = tf.keras.optimizers.Adam(lr=self.learning_rate, clipnorm=1.0)
+        strategy = tf.distribute.get_strategy()
+        if isinstance(strategy, tf.distribute.MirroredStrategy):
+            optimizer = tf.keras.optimizers.Adam(lr=self.learning_rate)
+        else:  # clipnorm is only supported in default strategy
```

Comment:
That's weird! Clipnorm is super important - do you know why?

Reply:
No idea and could not find online either.